js/base.js

```diff
@@ -1,3 +1,4 @@
+import * as os from "os";

```

Comment:
this will make the thing dependent on node we should be careful.. we may need to tune rollup.js

Reply:
This import was already being used on `kafka-consumer.js` and  `kafka-retry-consumer.js`

Reply:
So to change one I'd change all. That should maybe be done in a separate PR?

Reply:
Also, kafka consumers are using `fs` aswell.